src/components/views/settings/tabs/user/PreferencesUserSettingsTab.tsx

```diff
@@ -217,6 +237,20 @@ export default class PreferencesUserSettingsTab extends React.Component<IProps,
             // Only show the user onboarding setting if the user should see the user onboarding page
             .filter((it) => it !== "FTUE.userOnboardingButton" || showUserOnboardingPage(useCase));
 
+        const browserTimezoneLabel = `${_t("settings|preferences|default_timezone")} (${TimezoneHandler.shortBrowserTimezone()})`;
```

Comment:
The result of `TimezoneHandler.shortBrowserTimezone()` should be a variable of the default timezone translation.
The sentence can be structured differently in other languages.

```suggestion
        const browserTimezoneLabel = `_t("settings|preferences|default_timezone", { timezone:  TimezoneHandler.shortBrowserTimezone()})`
```

https://github.com/element-hq/element-web/blob/develop/docs/translating-dev.md#adding-variables-inside-a-string

Reply:
I don't know what I'm missing but with : 

```
"default_timezone": "Browser default (%(timezone))",
````

And 

```
_t("settings|preferences|default_timezone", { timezone: TimezoneHandler.shortBrowserTimezone()})
```

I have no interpolation and end up with `Browser default (%(timezone))`

Reply:
You missed the `s`. 

> Add the variable inside the string. The syntax for variables is %(variable)s. Please note the s at the end. The name of the variable has to match the previous used name.

`"default_timezone": "Browser default (%(timezone)s)",`